.tekton/verify-enterprise-contract-task-main-ci-pull-request.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && files.modified.exists(x, x.matches("verify-enterprise-contract.yaml"))
```

Comment:
I'm looking at https://cloud.google.com/secure-web-proxy/docs/cel-matcher-language-reference perhaps, endsWith or contains is a better option than matches. I'm not sure what kind of regex syntax will be used with matches.

Reply:
https://cloud.google.com/secure-web-proxy/docs/cel-matcher-language-reference#string-manipulation-operators